frame/support/src/storage/generator/mod.rs

```diff
@@ -58,7 +58,7 @@ mod tests {
 		#[pallet::config]
 		#[pallet::disable_frame_system_supertrait_check]
 		pub trait Config: 'static {
-			type BlockNumber;
+			type Block: sp_runtime::traits::Block;
```

Comment:
I think this file doesn't really need to change, it may have been due to a blanket copy-paste that you changed it, and then tried to fix it.

Reply:
These are actually the tests. Since the original config doesn't have BlockNumber anymore, I wanted to update the tests as well to extract it from the Block trait.